packages/components/src/internal/components/base/Grid.tsx

```diff
@@ -257,6 +259,7 @@ interface GridRowProps {
 const GridRow: FC<GridRowProps> = memo(({ columns, highlight, row, rowIdx }) => {
     // style cast to "any" type due to @types/react@16.3.14 switch to csstype package usage which does not declare
     // "textAlign" property correctly for <td> elements.
+    const rowObj = row.toObject();
```

Comment:
I'm skeptical about this. Why is this necessary now? Where is the column mapping not aligning with the data mapping?

Reply:
In some rare cases, there are 2 fieldKeys in the column set that really are for the same column, for example "case" vs "CASE". The getQueryDetails response will contain result for only one of them but row.get(column.index) is case sensitive.

Reply:
I agree this is not the ideal solution. I'll dig more and see if the duplicate columns can be avoided by the caller.

Reply:
I was able to revert this change by modifying getRequestString to prefer fieldkeys with the correct casing.

Reply:
Great, thanks for digging into this.